AEPLifecycle/Tests/LifecycleSessionTests.swift

```diff
@@ -170,7 +170,7 @@ class LifecycleSessionTests: XCTestCase {
         // test
         session.start(date: currentDate, sessionTimeout: sessionTimeoutInSeconds, coreMetrics: LifecycleMetrics())
 
-        let previousSessionInfo = LifecycleSessionInfo(startDate: Calendar.current.date(byAdding: .day, value: -8, to: currentDate),
+        let previousSessionInfo = LifecycleSessionInfo(startDate: currentDate.addingTimeInterval(-8*24*60*60), // currentDate - 8 days
```

Comment:
Using `Calendar` here will correct for daylight savings time changes. So, when the startDate is in standard time while the pauseDate is in daylight savings, the session time is "correctly" off by 3600 seconds. Better to just add/subtract from the time interval so the expected result is the same when the current time just past a daylight savings change.